src/Persistent/Pure.hs

```diff
@@ -245,6 +246,9 @@ runPersistExcept :: PersistExcept -> ServantErr
 runPersistExcept (PersistError500 msg)            = err500 { errBody = cs msg }
 runPersistExcept (PersistError404 msg)            = err404 { errBody = cs msg }
 runPersistExcept (PersistErrorNotImplemented msg) = err500 { errBody = cs msg }
+runPersistExcept (UserLoginInUse li) =
+    err500 { errBody = "user login in use: " <> cs (show li) }
+    -- FIXME: what's a good status code for 'login in use'?
```

Comment:
200? USers are know to reload pages and go back, etc. Otherwise, perhaps 403:  https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.4